charts/cert-manager-with-clusterissuer/templates/asleep-timeout.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: post-install
+  labels:
+    app: wait
+  annotations:
+    "helm.sh/hook": "post-install"
+    "helm.sh/hook-delete-policy": hook-succeeded, hook-failed
+spec:
+  containers:
+  - name: hook2-container
+    image: busybox
+    imagePullPolicy: IfNotPresent
+    command: ['sh', '-c', 'echo post-install hook Pod is running - hook-postinstall && sleep 120']
```

Comment:
could you please raise a jira to fix this in future and post the jira link here ?